content/docs/operating/integrations.md

```diff
@@ -78,6 +78,7 @@ For notification mechanisms not natively supported by the Alertmanager, the
   * [Signal](https://github.com/dgl/alertmanager-webhook-signald)
   * [SMS](https://github.com/messagebird/sachet): supports [multiple providers](https://github.com/messagebird/sachet/blob/master/examples/config.yaml)
   * [SNMP traps](https://github.com/maxwo/snmp_notifier)
+  * [Squadcast](https://support.squadcast.com/docs/prometheus): helps you get alerted via Phone call, SMS, Email and Push notifications and lets you take actions on those alerts.
```

Comment:
Please remove the text, just the link is enough.

Reply:
Updated and added DCO. Could you please approve?